new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleTraefikValidation.java

```diff
@@ -0,0 +1,685 @@
+
+    // check admin server pod exist
+    logger.info("Check for admin server pod {0} existence in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodCreated(adminServerPodName);
+
+    // check managed server pods exists
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Check for managed server pod {0} existence in namespace {1}",
+          managedServerPrefix + i, domainNamespace);
+      checkPodCreated(managedServerPrefix + i);
+    }
+
+    // check admin server pod is running
+    logger.info("Wait for admin server pod {0} to be ready in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodReady(adminServerPodName);
+
+    // check managed server pods are running
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Wait for managed server pod {0} to be ready in namespace {1}",
+          managedServerPrefix + i, domainNamespace);
+      checkPodReady(managedServerPrefix + i);
+    }
+
+    logger.info("Check admin service {0} is created in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkServiceCreated(adminServerPodName);
+
+    // check managed server services created
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Check managed server service {0} is created in namespace {1}",
+          managedServerPrefix + i, domainNamespace);
+      checkServiceCreated(managedServerPrefix + i);
+    }
+  }
+
+  @Test
+  @Order(2)
+  @DisplayName("Create an ingress per domain")
+  public void testCreatingIngress() {
+    // helm install parameters for ingress
+    ingressParam = new HelmParams()
+            .releaseName(domainUID + "-ingress")
+            .namespace(domainNamespace)
+            .chartDir(projectRoot + "/../kubernetes/samples/charts/ingress-per-domain");
+
+    assertThat(createIngress(ingressParam, domainUID, domainUID + ".org"))
+            .as("Test createIngress returns true")
+            .withFailMessage("createIngress() did not return true")
+            .isTrue();
+
+    // check the ingress is created
+    String ingressName = domainUID + "-traefik";
+    String cmd = String.format("kubectl get ingress %1s -n %2s", ingressName, domainNamespace);
+    assertThat(getExpectedResult(cmd, ingressName))
+            .as("ingress is created")
+            .withFailMessage("ingress is not created in namespace " + domainNamespace)
+            .isTrue();
+    logger.info("ingress is created in namespace {0}", domainNamespace);
+  }
+
+  @Test
+  @Order(3)
+  @DisplayName("verify the load balancer can reach the application ")
+  public void testSampleAppThroughLB() throws Exception {
+    String hostname = InetAddress.getLocalHost().getHostName();
+    String curlCmd = String.format("curl --silent --noproxy '*' -H 'host: %1s' http://%2s:30305/sample-war/index.jsp",
+        domainUID + ".org", hostname);
+    verifyLoadBalancer(curlCmd, 50);
```

Comment:
nitpick - i think you should call this `verifyIngressController` since we will likely later want tests that actually use load balancers, not ingress controllers, and those are really two different things

Reply:
changed the method name